composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "easyengine/service-command",
+    "description": "Shell to run helpful commands inside containers.",
```

Comment:
This description is of the shell-command I believe.

Reply:
Yes. Fixed